omero/users/clients-overview.rst

```diff
@@ -155,3 +155,11 @@ for a detailed list of supported formats.
     
     OMERO.importer
 
+OMERO.cli
+---------
+
+The |CLI| is a set of Python_ based system-administration, deployment and
```

Comment:
I'd be inclined to move the hyphen from "system administration" to "Python-based", just a suggestion though.